packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -894,6 +894,29 @@ const handleCfnFiles = (eventMap: EventMap, category: string, resource: string,
   });
 };
 
+const getNestedStackParameters = async (context: $TSContext, categories: Array<string>, amplifyMeta: any) => {
```

Comment:
```suggestion
const getNestedStackParameters = async (context: $TSContext, categories: Array<string>, amplifyMeta: $TSMeta) => {
```
Also, add return type to function signature